src/app/components/confirm-delete-profile/confirm-delete-profile.component.html

```diff
@@ -0,0 +1,14 @@
+<mat-card>
+  <mat-card-header>
+    <mat-card-title>Confirm deleting</mat-card-title>
+    <mat-card-subtitle [ngStyle]="{color: 'black'}">Do you really want to delete your profile?</mat-card-subtitle>
+  </mat-card-header>
+  <mat-card-content>
+    <p>All information associated with this profile will be permanently deleted.</p>
+    <p [ngStyle]="{color: 'red'}">This operation cannot be undone.</p>
+  </mat-card-content>
+  <mat-card-actions class="btn-confirm">
+    <button mat-button (click)="onNoClick()">Cancel</button>
```

Comment:
please remove click and just add mat-dialog-close

Reply:
Done